apps-rendering/src/components/Headline/GalleryHeadline.tsx

```diff
@@ -0,0 +1,69 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	from,
+	headline,
+	neutral,
+	remSpace,
+} from '@guardian/source-foundations';
+import { grid } from 'grid/grid';
+
+// ----- Component ----- //
+
+const styles = css`
+	${headline.medium({ fontWeight: 'bold' })}
+	background-color: ${neutral[7]};
+	color: ${neutral[100]};
+	padding: 0 ${remSpace[5]} ${remSpace[9]} 0;
+	${grid.between('centre-column-start', 'centre-column-end')}
```

Comment:
This could be `grid.column.centre` I think? (I've just noticed this is taken from the immersive headline, I'll tweak that too)

```suggestion
	${grid.column.centre}
```